sdk/storage/azblob/appendblob/client.go

```diff
@@ -338,6 +339,12 @@ func (ab *Client) CopyFromURL(ctx context.Context, copySource string, o *blob.Co
 	return blob.CopyFromURLResponse{}, errors.New("operation will not work on this blob type. CopyFromURL works only with block blob")
 }
 
+// GetSASURL is a convenience method for generating a SAS token for the currently pointed at append blob.
+// It can only be used if the credential supplied during creation was a SharedKeyCredential.
+func (ab *Client) GetSASURL(permissions sas.BlobPermissions, expiry time.Time, o *blob.GetSASURLOptions) (string, error) {
```

Comment:
This has been added to blockblob and pageblob. Do we need the same for AppendBlob as well?

Reply:
This has been added to all specialized clients - including Append Blob.
See AppendBlob implementation of GetSASUrl here - https://github.com/Azure/azure-sdk-for-go/blob/b268345f3c913cfcd8d6138b15e3e1445ec9dabe/sdk/storage/azblob/appendblob/client.go#L344